manifests/gcp_marketplace/deployer/init_action.sh

```diff
@@ -62,9 +61,24 @@ NAME="$(/bin/print_config.py \
 NAMESPACE="$(/bin/print_config.py \
     --xtype NAMESPACE \
     --values_mode raw)"
+NEED_DEFAULT_BUCKET="$(/bin/print_config.py \
+    --xtype STRING \
+    --values_mode raw)"
```

Comment:
Where are we specifying `needDefaultBucket` field in this `print_config.py` call?

Reply:
Good question. Currently it's actually matching according to the --xtype, which is supposed to be unique in a schema.yaml instance.

cc'ed you in a thread with marketplace team and I will ask them over there

Reply:
Unfortunately there's no easy way to access arbitrary field value in schema.yaml according the key/field name. Using `--xtype=STRING` is a workaround.

That also means this approach won't work if we want to condition the logic on `managedstorage.gcsBucketName`. I think we have two options here:

1. Put a PR to change the marketplace deployer utility.

2. Use a piece of shell script to parse the yaml format printed out by `print_config.py -o yaml`. 

I slightly prefer 2 because it's faster and not subject to MKP image release, etc, but it's also more brittle. WDYT?

Reply:
Agree, @numerology yq is a good tool for this purpose: https://mikefarah.gitbook.io/yq/